static/app/views/alerts/incidentRules/triggers/chart/index.tsx

```diff
@@ -72,8 +72,19 @@ const TIME_PERIOD_MAP: Record<TimePeriod, string> = {
 };
 
 /**
- * If TimeWindow is small we want to limit the stats period
- * If the time window is one day we want to use a larger stats period
+ * Just to avoid repeating it
+ */
+const MOST_TIME_PERIODS: TimePeriod[] = [
+  TimePeriod.ONE_DAY,
+  TimePeriod.THREE_DAYS,
+  TimePeriod.SEVEN_DAYS,
+  TimePeriod.FOURTEEN_DAYS,
+  TimePeriod.THIRTY_DAYS,
+];
```

Comment:
`as const`?

Reply:
nah as const will mess up the `TimePeriod[]` that i want, could be reasonly though